lib/haml_lint/linter/alt_text.rb

```diff
@@ -0,0 +1,11 @@
+module HamlLint
+  class Linter::AltText < Linter
+    include LinterRegistry
+
+    def visit_tag(node)
+      if node.tag_name == 'img' && !node.has_hash_attribute?(:alt)
```

Comment:
This technique is specifically related to `img` tags (http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/H37). There are other requirements to meet the guideline I linked to in the PR, but this is an easy test that moves towards that goal. I can make the guideline name more specific, like `ImgAltText` if that's more clear.